src/main/java/com/exasol/projectkeeper/ProjectFilesFitter.java

```diff
@@ -0,0 +1,75 @@
+package com.exasol.projectkeeper;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.StandardCopyOption;
+import java.util.List;
+
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugin.logging.Log;
+
+/**
+ * This class fits the projects file structure.
```

Comment:
fits?

Reply:
it makes the structure fit
Is it the wrong English form?

Reply:
It just sounds somehow weird. I would say `shape the structure` or `order the structure`. What do you think?

Reply:
```suggestion
 * This class shape the project's file structure.
```